src/USER-REAXC/pair_reaxc.cpp

```diff
@@ -387,7 +387,7 @@ void PairReaxC::init_style( )
   neighbor->requests[irequest]->newton = 2;
   neighbor->requests[irequest]->ghost = 1;
 
-  cutmax = MAX3(control->nonb_cut, control->hbond_cut, 2*control->bond_cut);
+  cutmax = MAX3(control->nonb_cut, control->hbond_cut, control->bond_cut);
```

Comment:
Please note, that the same change would need to be made in USER-OMP/pair_reaxc_omp.cpp as it overrides init_style().

Reply:
If the pull request is accepted, I can make the change in the USER-OMP version of ReaxFF.

Reply:
You have it backwards. We won't merge a pull request with known inconsistencies.

Reply:
Of course, I've made the change.